### PR TITLE
Travis test

### PR DESCRIPTION
**Pull request by chandra92**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1 +1,10 @@
 language: node_js
+services: redis-server
+env: DB=postgres
+before_script: psql -c 'create database testingdatabase;' -U postgres && sleep 1 && psql -U postgres -d testingdatabase -a -f tools/pg/sbcontent.sql && sleep 1 && psql -U postgres -d testingdatabase -a -f tools/pg/sbentity.sql
+
+addons:
+ postgresql: "9.3"
+
+notifications:
+ email: false
```
Comment (satya164) [inline .travis.yml]: Shouldn't email be enabled so that we get email when tests fail?

File: storage/test/queries-test.js [hunk 2]
```diff
@@ -1,4 +1,9 @@
 /* jshint mocha: true */
```
Comment (satya164) [inline storage/test/queries-test.js]: You can use /* eslint-env mocha */

File: .travis.yml (hunk 1 above)
Comment (aravindet) [inline .travis.yml]: Yes, these should replace the nightly builds we are currently doing, and send emails.

**Comment by aravindet** [comment]
Also, why replace all the loop variables i with j? Did you have any issues because of that? On quick overview it looks like they should not clash?

**Comment by chandra92** [comment]
Yes, emails will be enabled once it is complete.

*Thank You*
*Chandrakant*

**Pull request closed without merging**